template.rb

```diff
@@ -241,6 +241,11 @@ def render_file(filename)
     end
   end
 
+  initializer 'open_uri.rb', <<-CODE
```

Comment:
I don't have familiarity with templates. Would it be easy to describe what this is doing? I have a few questions:
1. I see reference to `open_uri.rb` but I don't see that file anywhere in this repo. Is this line saying create this file in the `initializers` folder and that file contains the one line of `require "open-uri"`?
1. What is this syntax with ` <<-CODE` followed by another `CODE`? Is it pushing code into `open-uri.rb` with the `<<`?
1. Does that mean that `CODE` is being used to define a block?

Reply:
1. I followed the Rails Guide on [application templates](https://edgeguides.rubyonrails.org/rails_application_templates.html#vendor-lib-file-initializer-filename-data-nil-block). This creates a new file in the `initializers/` folder. I had issues creating the file the other ways. I'll try again once master is up to speed.

1. This is heredoc syntax. I wrote a [blog post](https://medium.com/@jelaniwoods/what-is-a-heredoc-7a9515ad9848) about it. 

1.  Kinda

Reply:
Awesome! Thanks!